### ISSUE TITLE
Non-text files will not resolve into a 404 error page

### ISSUE THREAD

**Issue by joshbruce** [8fold/site-joshbruce.com#186]
**Describe the bug**

If an invalid image file is entered as the URL, it does not result in a 404 error page.

**To Reproduce**

https://joshbruce.com/sitemap.jpg

**Expected behavior**

When any resource is entered and doesn't exist, the 404 error page should be displayed.
